scripts/handle-user-data

```diff
@@ -33,6 +35,10 @@ mount_data_partition() {
     mkdir -p /mnt/$PARTNAME/var/lib/boot2docker/etc/docker
     ln -s /mnt/$PARTNAME/var/lib/boot2docker/etc/docker /etc/docker
 
+    mkdir -p /mnt/$PARTNAME/var/lib/origin
+    mkdir -p /var/lib/origin
+    mount --bind /mnt/$PARTNAME/var/lib/origin/ /var/lib/origin/
```

Comment:
Let's do a bind mount for Docker as well then. WDYT? I find it odd to use symbolic link once and bind mount the other time.